group-coordinator/src/main/java/org/apache/kafka/coordinator/group/consumer/TargetAssignmentBuilder.java

```diff
@@ -393,16 +397,16 @@ private Assignment newMemberAssignment(
         }
     }
 
-    static AssignmentMemberSpec createAssignmentMemberSpec(
+    // private for testing
+    static MemberSubscriptionSpecImpl createMemberSubscriptionSpecImpl(
         ConsumerGroupMember member,
-        Assignment targetAssignment,
-        TopicsImage topicsImage
+        TopicsImage topicsImage,
+        Assignment memberAssignment
```

Comment:
nit: Let's revert back to the previous order in order to reduce the noise in the diff.